pandas/core/series.py

```diff
@@ -1279,7 +1279,15 @@ def multi(values, qs):
         return self._maybe_box(lambda values: multi(values, q), dropna=True)
 
     def ptp(self, axis=None, out=None):
-        return _values_from_object(self).ptp(axis, out)
+        """
```

Comment:
This should mirror the signature of ``max/min`` (e..g ``axis/level/skipna/numeric_only``). remove the ``.out`` arg.